tests/console/ant.pm

```diff
@@ -91,8 +91,8 @@ sub run {
     assert_script_run "cd $dir";
     assert_script_run "mkdir -p src/test";
     assert_script_run "mkdir -p build/classes";
-    assert_script_run "echo '$java_hello_world' >> src/test/Hello.java";
-    assert_script_run "echo '$build_file_xml' >> build.xml";
+    script_output("echo '$java_hello_world' >> src/test/Hello.java");
```

Comment:
What's the purpose of this change?

Reply:
On the top you will see the content of `$java_hello_world`, it is much better and stable for longer complex text use script_output, it will execute it as script, than (assert_)script_run which will execute it as command and possibly fail.
http://10.100.12.155/tests/15694#step/ant/23

Reply:
Seems like `assert_script_run` does it too: http://10.100.12.155/tests/15694#step/ant/17

Looking at the code, `script_output` is more complex than `assert_script_run` and in some cases even uses `assert_script_run`.

It just seems weird to me to use `script_output` for something which doesn't output anything.
If there is a specific issue with `assert_script_run`, that should probably be fixed in os-autoinst instead.

Reply:
Too much input for my taste, I would rather prefer to upload the code from `data` folder.

Reply:
dropped ant, will check it out later